client/js/tract-details.js

```diff
@@ -294,7 +298,7 @@ var trpanels = d3.select("#trackdetails").selectAll("svg").data(trackdata);
   function brushEnd() {
 	  brushing = false;
   }
-}
+
```

Comment:
That extra closing bracket should be moved here so that all the brushing functions are in the same scope as the d3.brush.